apiaudio/helper_classes.py

```diff
@@ -17,32 +17,48 @@ class CreatableResource(APIRequest):
     def create(cls, **params):
         return cls._post_request(json=params)
 
+
 class DeletableResource(APIRequest):
     @classmethod
     def delete(cls, **args):
 
-        if cls.OBJECT_NAME == 'media':
-            if 'mediaId' not in args:
-                raise Exception('mediaId parameter is required')
-            
-            return cls._delete_request(path_param=cls.audio_resource_path, request_params=args)
+        if cls.OBJECT_NAME == "media":
```

Comment:
Im guessing this is all coming from black/linter? 

 Im guessing as this is the first time its run it probably should be a separate PR otherwise it makes it a little harder to review :D

Reply:
well, it's messy just once, from now on the whole source code is compliant with the standard :)